NextTime.yaml

```diff
@@ -11,3 +11,4 @@ suggestions:
   - Building codebase (B script)
   - CMakeLists
   - Making a module
+  - Making a role
```

Comment:
If you're interested in learning more about roles, there is [this NUbook page](https://nubook.nubots.net/system/foundations/overview#roles). I'm also planning to extend the Getting Started page to include a tutorial that will include roles :smile:, and we could do a workshop on it. 
Nice suggestion!